YoCodeAutomatedTests/HelpMessageTest.cs

```diff
@@ -3,21 +3,34 @@
 using YoCode;
 using System.IO;
 using System;
+using Microsoft.Extensions.Configuration;
 
 namespace YoCodeAutomatedTests
 {
     public class HelpMessageTest
     {
+        private string TestPath;
+        private string dllPath;
+        public static IConfiguration Configuration;
+ 
+        public HelpMessageTest()
+        {
+            var builder1 = new ConfigurationBuilder().SetBasePath(Directory.GetCurrentDirectory()).AddJsonFile("testappsettings.json");
+            Configuration = builder1.Build();
```

Comment:
This should be done in just one place. Every test is going to need it, so it would make sense to do it once in a place where every test has access to it. Make a class called something like `TestConfiguration` and put this code in there.